client/e2e/setup/global.ts

```diff
@@ -0,0 +1,143 @@
+/**
+ 📖
+ The globalSetup option in playwright.config.js allows you to specify a JavaScript file that will be executed ONCE before all test suites.
+ This file is useful for setting up any global resources, initializing databases, or creating authentication storageStates
```

Comment:
If I understand this correctly (and really not sure I do haha), this means that every time we run the test suite, we will for real login using keycloak (and then that login info is saved for use in later tests). I'd like to get @dleard's thoughts on this--on CIIP/CIF/etc. we used cypress and mocked login, and I'm not sure if that was because it was important to not hammer keycloak with requests (cypress didn't store anything though, so it would have been every test, and here's it's only every suite)